packages/integration-tests/suites/tracing/browsertracing/backgroundtab-pageload/test.ts

```diff
@@ -9,9 +9,11 @@ sentryTest('should finish pageload transaction when the page goes background', a
 
   await page.goto(url);
 
-  void page.click('#go-background');
+  const pageloadTransactionPromise = getFirstSentryEnvelopeRequest<Event>(page);
 
-  const pageloadTransaction = await getFirstSentryEnvelopeRequest<Event>(page);
+  await page.click('#go-background');
+
+  const pageloadTransaction = await pageloadTransactionPromise;
```

Comment:
Can we just `Promise.all` (kicking off the page click first?)?

Reply:
That is inherently gonna produce the race condition again that I want to remove with this change. Listen first -> then the interaction -> wait for interaction outcome. Mixing step 1 and 2 is dangerous.